fendermint/vm/topdown/src/sync.rs

```diff
@@ -340,7 +340,10 @@ async fn last_recorded_data(
             match atomically(|| provider.first_non_null_parent_hash(height)).await {
                 None => unreachable!("should have last committed finality at this point"),
                 Some(hash) => {
-                    tracing::info!("First non null parent hash: {hash:02x?} at height: {height}");
+                    tracing::info!(
```

Comment:
Can we add also an info trace when we commit the finality? That way we know the last non-null parent and what is being committed in the chain as final.

Reply:
Structured logging again would help keep logs consistent; for example if we use `block_height` consistently then a log aggregator would turn that into a searchable field and you can see everything that happened in a block.